src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -339,9 +344,41 @@ class about extends viewModelBase {
         shell.openFeedbackForm();
     }
 
+    private getConnectivityToLicenseServer() {
+        this.spinners.connectivityToLicenseServer(true);
+        return new getConnectivityToLicenseServerCommand()
+            .execute()
+            .done((connectionResult: Raven.Server.Web.Studio.LicenseHandler.ConnectivityToLicenseServer) =>
+                this.connectedToLicenseServer(connectionResult.StatusCode !== "OK"))
+            .always(() => this.spinners.connectivityToLicenseServer(false));
+    }
+
+    refreshConnectivityToLicenseServer() {
```

Comment:
why we need both getConnectivityToLicenseServer and refreshConnectivityToLicenseServer

Reply:
no need - removed the 'refresh'